docs/user/datastores/query_config.rst

```diff
@@ -36,6 +36,13 @@ To set a hint through GeoServer, modify your query URL to use the ``viewparams``
 
 Hint values will be converted into the appropriate types. See below for available hints and their accepted values.
 
+Command-Line Tools
+^^^^^^^^^^^^^^^^^^
+
+When exporting features through the GeoMesa command line tools, query hints can be set using the `--hints` parameter.
+Hints should be specified in the form `key1=value1;key2=value2`. Hints are converted to the appropriate type
+according to the conventions for GeoServer hints.
```

Comment:
i actually meant geoserver, if that's what you mean - there is a section on how a string gets converted into the appropriate hint type for geoserver